app/ui/screens/events/EventScreen.js

```diff
@@ -379,9 +380,26 @@ costs of €{{fine}}. Also note that you will be unable to re-register.',
 
   render() {
     const {
-      status, loading, openMaps, data, t, openAdmin, openUrl,
+      status, loading, openMaps, data, t, openAdmin, openUrl, addToCalendar,
     } = this.props;
 
+    const addCalendarButton = (
+      <TouchableOpacity
+        onPress={() => addToCalendar(
+          this.props.data.title,
+          this.props.data.location,
+          this.props.data.start,
+          this.props.data.end,
+        )}
+      >
+        <CommunityIcon
+          name="calendar-export"
```

Comment:
This button is not showing up ![Screenshot_20201011-155543.jpg](https://user-images.githubusercontent.com/1799914/95680513-a5f4ec00-0bda-11eb-95d9-d019acbd8b66.jpg)

Reply:
Hmm did not see that, on iOS this works. Will look at it.

Reply:
The fonts were not linked, should be fixed now.